### PR TITLE
Process by cell for higher resolution models

### PR DESCRIPTION
**Pull request by ppapasani1**
Add PET filtering
Fix defense
Clean up

### CODE REVIEW

File: climateE2E/climateE2EMain.R [hunk 1]
```diff
@@ -43,18 +43,23 @@ source(file.path(MODULE_DIR,"e2e_logger.R"))
...
 if(args$runType == "all"){
   source(file.path(MODEL_DIR,toupper(perilRegion),paste0(tolower(perilRegion),"_hazard.R")))
+  writeData(hazard, config$OutFolder, "hazard")
   source(file.path(MODEL_DIR,toupper(perilRegion),paste0(tolower(perilRegion),"_vuln.R")))  
+  writeData(damage, config$OutFolder, "damage")
 }else if(args$runType == "hazard"){
   source(file.path(MODEL_DIR,toupper(perilRegion),paste0(tolower(perilRegion),"_hazard.R")))
+  writeData(damage, config$OutFolder, "hazard")
 }else if(args$runType == "damage"){
   source(file.path(MODEL_DIR,toupper(perilRegion),paste0(tolower(perilRegion),"_vuln.R")))
```
Comment (sthupakula-rms) [inline climateE2E/climateE2EMain.R:61]: Can this run by itself without running hazard?

File: climateE2E/Module/outputManager.R [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
+writeCsv <- function(data, outputFile){
+  log_info("Writing CSV file with {nrow(data)} to {outputFile}")
+  fwrite(data,outputFile,row.names = F, quote = T)
```
Comment (sthupakula-rms) [inline climateE2E/Module/outputManager.R:4]: why `quote = T`?

File: climateE2E/Module/outputManager.R [hunk 3]
```diff
@@ -0,0 +1,18 @@
+
+writeCsv <- function(data, outputFile){
+  log_info("Writing CSV file with {nrow(data)} to {outputFile}")
```
Comment (sthupakula-rms) [inline climateE2E/Module/outputManager.R:3]: log_info("Writing CSV file with {nrow(data)} **rows** to {outputFile}")

File: climateE2E/Module/outputManager.R [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  fwrite(data,outputFile,row.names = F, quote = T)
+}
+
+writeParquet <- function(data, outputFile){
+  log_info("Writing Parquet file with {nrow(data)} to {outputFile}")
```
Comment (sthupakula-rms) [inline climateE2E/Module/outputManager.R:8]: same as above

**Review by sthupakula-rms — APPROVED** [review]

File: climateE2E/climateE2EMain.R (hunk 1 above)
Comment (ppapasani1) [reply climateE2E/climateE2EMain.R:62]: No. kept as a place holder for future partitioning work. Any way added it now.

File: climateE2E/Module/outputManager.R (hunk 2 above)
Comment (ppapasani1) [reply climateE2E/Module/outputManager.R]: Fixed

File: climateE2E/Module/outputManager.R (hunk 3 above)
Comment (ppapasani1) [reply climateE2E/Module/outputManager.R]: Fixed

File: climateE2E/Module/outputManager.R (hunk 4 above)
Comment (ppapasani1) [reply climateE2E/Module/outputManager.R]: Fixed

**Pull request merged**